src/components/Time/ErrorManager.vue

```diff
@@ -248,7 +248,7 @@ export default {
           this.$root.$emit("cancelExpired");
           this.$root.$emit("removeLayer", layer);
           this.expiredSnackBarMessage = this.$t("UnhandledError");
-          console.log(e);
+          console.log(response);
```

Comment:
Is this for debugging? or an error? Use `console.debug()` or `console.error()`.

Reply:
Changed for console error and added small message